nipype/info.py

```diff
@@ -143,17 +143,14 @@ def get_nipype_gitversion():
     "pydot>=%s" % PYDOT_MIN_VERSION,
     "pydotplus",
     "python-dateutil>=%s" % DATEUTIL_MIN_VERSION,
+    "rdflib @ git+https://github.com/rdflib/rdflib.git@master",
```

Comment:
this would go into `setup.py`... AFAIK ideally such restrictions are to be stated only in `requirements.txt` and here only minimal or problematic versions to be listed.

Reply:
Yeah, this is a temporary measure to ensure that we're testing against latest. rdflib 5.0 is due to be released this weekend, and I'll switch it then.